Description: Make the build reproducible
Author: Chris Lamb <lamby@debian.org>
Last-Update: 2018-02-25

--- gr-gsm-0.41.2.orig/lib/CMakeLists.txt
+++ gr-gsm-0.41.2/lib/CMakeLists.txt
@@ -44,7 +44,7 @@ endmacro()
 # Handle the generated constants
 ########################################################################
 execute_process(COMMAND ${PYTHON_EXECUTABLE} -c
-    "import time;print time.strftime('%a, %d %b %Y %H:%M:%S', time.gmtime())"
+    "import os,time,datetime; print datetime.datetime.utcfromtimestamp(int(os.environ.get('SOURCE_DATE_EPOCH', time.time()))).strftime('%a, %d %b %Y %H:%M:%S')"
     OUTPUT_VARIABLE BUILD_DATE OUTPUT_STRIP_TRAILING_WHITESPACE
 )
 message(STATUS "Loading build date ${BUILD_DATE} into constants...")
